README.md

```diff
@@ -1,6 +1,6 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains Norik's protfolio and SPS projects you'll work on throughout SPS.
```

Comment:
Nit: You can keep this string as "This repo contains Norik's protfolio and SPS projects."